libs/utils/preflight-notification.js

```diff
@@ -0,0 +1,99 @@
+import { hasPreflightFailures, executePreflightChecks } from '../blocks/preflight/checks/preflightExecutor.js';
+import { loadStyle, getConfig } from './utils.js';
+
+let preflightNotificationDismissed = false;
+
+function openPreflightPanel() {
+  const sk = document.querySelector('aem-sidekick, helix-sidekick');
+  if (sk) {
+    sk.dispatchEvent(new CustomEvent('custom:preflight', { bubbles: true }));
+  }
+}
+
+async function createPreflightNotification() {
+  const existingNotification = document.querySelector('.milo-preflight-overlay');
+  if (existingNotification) {
+    existingNotification.remove();
+  }
```

Comment:
nit: Usually, in Milo, we write single-statement if block in one line. Makes the code neater
```suggestion
  if (existingNotification) existingNotification.remove();
```